docs/install.md

```diff
@@ -4,70 +4,88 @@ nav_order: 2
 
-<https://github.com/containers/bootc/blob/main/docs/install.md#using-bootc-install-to-disk---via-loopback>
+### The dedicated cloud-init image
 
-to generate a raw disk image from the default container base image, or your own
-without any external tooling.
+Many people who just want to "try things out" will find it easiest to start
+with
+[the cloud image](https://gitlab.com/bootc-org/centos-bootc-layered/-/tree/main/cloud).
+It's a separate container image because cloud-init does not work on every deployment
+target, and it also serves as an effective demonstration of layering.
 
-If you choose not to include SSH keys or other credentials directly in your image,
-a useful pattern can often be to use [systemd credentials](https://systemd.io/CREDENTIALS/)
-to inject a SSH key for root.  The above page has this example for qemu:
+## Production-oriented physical installation
 
-```bash
--smbios type=11,value=io.systemd.credential.binary:tmpfiles.extra=$(echo "f~ /root/.ssh/authorized_keys 600 root root - $(ssh-add -L | base64 -w 0)" | base64 -w 0)
+This project uses the same
+[Anaconda](https://anaconda-installer.readthedocs.io/en/latest/intro.html)
+installer as the package-based CentOS.  Here's an example kickstart:
+
+```text
+# Basic setup
+text
+network --bootproto=dhcp --device=link --activate
+# Basic partitioning
+clearpart --all --initlabel --disklabel=gpt
+reqpart --add-boot
+part / --grow --fstype xfs
+
+# Here's where we reference the container image to install - notice the kickstart
+# has no `%packages` section!  What's being installed here is a container image.
+ostreecontainer --url quay.io/centos-bootc/centos-bootc:stream9 --no-signature-verification
+
+firewall --disabled
+services --enabled=sshd
+
+# Only inject a SSH key for root
+rootpw --iscrypted locked
+sshkey --username root "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOQkQHeKan3X+g1jILw4a3KtcfEIED0kByKGWookU7ev walters+2015-general@verbum.org"
+reboot
 ```
 
-## Installation using Anaconda
+## Production-oriented cloud virtualization
 
-Tools like
-[Anaconda](https://anaconda-installer.readthedocs.io/en/latest/intro.html)
-support injecting configuration at image installation time, such as SSH keys and
-passwords. This means that in contrast to what was said just before, it's
-possible to directly install (and update from) an "unconfigured base image"
-provided by this project.
+### Generating AMIs and qcow2 (and more)
 
-Because a current development target for this project is [Fedora ELN](https://docs.fedoraproject.org/en-US/eln/),
-which includes the latest support for `bootupd`, it's recommended to use
-that ISO at this time.  The support for `ostreecontainer` does not
-yet exist in CentOS Stream 9.
+The [bootc-image-builder tool](https://github.com/osbuild/bootc-image-builder)
+which supports `.qcow2` usable in OpenStack/libvirt etc. also supports generating
+Amazon Machine Images and other production-oriented IaaS formats.
```

Comment:
Amazon Machine Images (ami), Native Apple Hypervisor Images (raw) and other production-...